samples/ClientSample/HubSample.cs

```diff
@@ -92,5 +87,26 @@ internal static void Register(CommandLineApplication app)
             }
             return 0;
         }
+
+        private static async Task<HubConnection> ConnectAsync(string baseUrl)
+        {
+            // Keep trying to until we can start
+            while (true)
+            {
+                var connection = new HubConnectionBuilder()
+                                .WithUrl(baseUrl)
+                                .WithConsoleLogger(LogLevel.Trace)
+                                .Build();
+                try
+                {
+                    await connection.StartAsync();
+                    return connection;
+                }
+                catch (Exception)
+                {
+                    await Task.Delay(1000);
```

Comment:
`Console.WriteLine("Retrying...")`?
Also should we try a different transport when retrying?

Reply:
This is mostly useful for when you launch the C# client and haven't started the server yet, don't think we should be changing transports.

Reply:
Yea, I just wanted to make that work so it would be easy to test.